packages/playwright-test/src/fixtures.ts

```diff
@@ -222,6 +222,15 @@ export class FixturePool {
         if (options.scope === 'worker' && disallowWorkerFixtures)
           throw errorWithLocations(`Cannot use({ ${name} }) in a describe group, because it forces a new worker.\nMake it top-level in the test file or put in the configuration file.`, { location, name });
 
+        // Overriding option with "undefined" value means setting it to the default value
+        // from the original declaration of the option.
+        if (fn === undefined && options.option && previous) {
```

Comment:
`fn` is a bit weird in this context, consider a more descriptive name `valueOrFn` maybe?